github/tests/Framework.py

```diff
@@ -241,6 +243,7 @@ def setUp(self):
             self.oauth_token = "oauth_token"
             self.client_id = "client_id"
             self.client_secret = "client_secret"
+            self.jwt = "jwt"
```

Comment:
Shouldn't you add a `activateJwtAuthMode` in this file as well? Since this PR adds support for authenticating with JWT, we should support recording tests through it right?

Reply:
@sfdye thanks for the suggestion, I added the mode and verified that it works.